tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -515,6 +551,7 @@
                 Field(
                     item="26AVII",
                     name="OTHER_SANCTION",
+                    friendly_name="other, sanction",
```

Comment:
is the comma intended?

Reply:
yes, based on the pdf file for field names